test/numerical/TestConv.cpp

```diff
@@ -0,0 +1,147 @@
+#include <algorithm>
+#include <cmath>
+#include <iostream>
+#include <random>
+#include <rapidcheck.h>
+#include <string>
+#include <vector>
+
+#include "mlir/IR/Module.h"
+#include "llvm/Support/FileSystem.h"
+
+#include "src/Dialect/ONNX/ONNXOps.hpp"
+#include "src/MainUtils.hpp"
+#include "src/Runtime/ExecusionSession.hpp"
+
+using namespace std;
+
+int main() {
```

Comment:
Could you add a comment on how this is invoked? I expected a loop going over various N, C, H,.... values; here I see only one. I would add a few lines on how main is invoked (repetitively, with varying values?) and if so, where is the driver of this function.

thanks

Reply:
Added a documentation page on testing to explain how numerical tests are structured in general. I also added examples of exhaustive testing for a small range of values.